spec/chromium-spec.js

```diff
@@ -1515,11 +1515,7 @@ describe('font fallback', () => {
     try {
       await w.loadURL(`data:text/html,${html}`)
       w.webContents.debugger.attach()
-      const sendCommand = (...args) => new Promise((resolve, reject) => {
-        w.webContents.debugger.sendCommand(...args, (e, r) => {
-          if (e) { reject(e) } else { resolve(r) }
-        })
-      })
+      const sendCommand = (...args) => w.webContents.debugger.sendCommand(...args)
```

Comment:
Can probably just call `w.webContents.debugger.sendCommand` directly, or if that's too verbose then `const {debugger} = w.webContents; debugger.sendCommand(...)`